zulipterminal/ui_tools/boxes.py

```diff
@@ -97,6 +97,15 @@ def __init__(self, view: Any) -> None:
         # Prioritizes autocomplete in message body
         self.recipient_user_ids: List[int]
 
+        # Marks if the write box formed is involves [EDIT_MESSAGE: e] key
+        # Only used for private messages(for now)
+        self.edit_keypressed: bool
+        self.edit_keypressed = False
```

Comment:
The edit state is encoded in `msg_edit_state` - does this give more information?

Reply:
I didn't think of that...nope, it does not give any other information. It seems that the same work can be done using `msg_edit_state`(I ran the code replacing `edit_keypressed` with `msg_edit_state` and it worked fine.)
So I will remove the `edit_keypressed` part and use `msg_edit_state`  instead :)